paimon-core/src/main/java/org/apache/paimon/operation/AppendOnlyFileStoreRead.java

```diff
@@ -66,7 +68,9 @@ public class AppendOnlyFileStoreRead implements FileStoreRead<InternalRow> {
     private final TableSchema schema;
     private final FileFormatDiscover formatDiscover;
     private final FileStorePathFactory pathFactory;
-    private final Map<FormatKey, BulkFormatMapping> bulkFormatMappings;
+    private final Map<FormatKey, Triple<TableSchema, List<Predicate>, BulkFormatMapping>>
```

Comment:
Maybe it is better to add `TableSchema, List<Predicate>` to `BulkFormatMapping`?